2017-10/agenda.md

```diff
@@ -0,0 +1,47 @@
+# Node.js Collaborator Summit, 2017-10-06 - 2017-10-07, Vancouver, BC, Canada
+
+**This is a DRAFT proposed agenda**
+
+## 2017-10-06 Day One
+
+|    Time Slot    | Room |      Session     |        Description       |
+| --------------- | ---- | ---------------- | ------------------------ |
+|  1:30p - 3:00p  | TBD  | Core Plenary     | All-Collaborator Plenary |
+|  3:00p - 5:00p  | TBD  | Working Sessions | Working Sessions         |
+
+## 2017-10-07 Day Two
+
+|    Time Slot    | Room |      Session     |        Description       |
+| --------------- | ---- | ---------------- | ------------------------ |
+|  9:00a - 10:30a | TBD  | CommComm Plenary | CommComm Plenary Session |
+|  9:00a - 12:00a | TBD  | Working Sessions | Working Sessions         |
+| 12:00p - 1:00p  | TBD  | Lunch            | Lunch                    |
+|  1:00p - 3:00p  | TBD  | Working Sessions | Working Sessions         |
+|  3:00p - 5:00p  | TBD  | Closing Plenary  | All-Collaborator Plenary |
+
+## Working Group Sessions
```

Comment:
I would call them "sessions", and specify that also teams can meet and discuss things.